components/__tests__/UnitInput.test.ts

```diff
@@ -0,0 +1,163 @@
+import { mount } from '@vue/test-utils';
+// @ts-ignore
+import UnitInput from '@/components/form/UnitInput.vue';
+// @ts-ignore
+import LabeledInput from '@/components/form/LabeledInput.vue';
+import { UNITS } from '../../utils/units';
+
+describe('unitinput', () => {
```

Comment:
I see that most of the tests have multiple assertions for the same tests.

It's usually recommended to adhere to an AAA pattern, so every test is isolated and aimed to a single scope. It may still have multiple expect cases if the rest of the test is the same.

Note:
AAA stands for Arrange, Act and Assert.
In the Arrange you prepare the test, e.g. set properties to a component or declare variables.
Act is basically when you call the function, most of the time.
Assertion is where you write what to expect from the test